### DISCUSSION TITLE
小程序加载pag，40多帧的时候突然报错，卡顿到卡死

### DISCUSSION THREAD

**Discussion by qing-ke** [Tencent/libpag#2344] [Q&A]
<img width="363" alt="B24CCEC2-94F3-4BBF-A118-94B63BFD7CC4" src="https://github.com/Tencent/libpag/assets/14007830/0953e817-4758-4eed-8f70-fc005b2ca9ec">
demo使用的是github里的小程序demo。
<img width="702" alt="56A523F7-6DD1-4DAA-BE8F-9F158BE2F471" src="https://github.com/Tencent/libpag/assets/14007830/9bef1c90-39d8-4aa1-946b-131cbf3e9e93">

**Comment by CodeJhF** [comment]
之前请你补充的复现信息（demo / 版本号 / 机型 / 日志）一直没有收到，缺少这些信息无法继续定位，先关闭本条。
如果问题还在，欢迎补齐上述信息后回复或重开，我们会优先跟进。（stale-close）
